packages/react-cosmos-webpack/src/normalize-modules.js

```diff
@@ -0,0 +1,30 @@
+const {
+  default: importModule
+} = require('react-cosmos-utils/lib/import-module');
+const {
+  default: importComponent
+} = require('react-cosmos-utils/lib/import-component');
+
+export const normalizeComponents = components =>
+  Object.keys(components).reduce(
+    (acc, next) => ({
+      ...acc,
+      [next]: importComponent(components[next], next)
+    }),
+    {}
+  );
+
+export const normalizeFixtures = fixtures =>
+  Object.keys(fixtures).reduce((acc, next) => {
```

Comment:
❤️  how "functional" this function is 😄

Reply:
Not really proud of `acc2` and `next2` tho 😅